website/content/en/docs/amazon-sagemaker-integration/sagemaker-components-for-kubeflow-pipelines.md

```diff
@@ -10,10 +10,91 @@ These components integrate SageMaker with the portability and orchestration of K
 
 ## Available components
 
-You can create a Kubeflow Pipeline built entirely using SageMaker components, or integrate individual components into your workflow as needed. Available Amazon SageMaker components can be found in the [Kubeflow GitHub repository](https://github.com/kubeflow/pipelines/tree/master/components/aws/sagemaker).
+You can create a Kubeflow Pipeline built entirely using SageMaker components, or integrate individual components into your workflow as needed. Available Amazon SageMaker components can be found in the [Kubeflow Pipelines GitHub repository](https://github.com/kubeflow/pipelines/tree/master/components/aws/sagemaker).
 
-## Tutorials
+Currently, we support two versions of these SageMaker components - the traditional v1 components and the ACK Based v2 components. The Sample included with Kubeflow is currently based off of the v2 Training Component. However, you can just as easily configure and use all others by referring to our more detailed guides over at the pipelines repo - 
+1. [AWS Samples for SageMaker Kubeflow Pipeline Components](https://github.com/kubeflow/pipelines/tree/master/samples/contrib/aws-samples)
+2. We have also added a new sample which allows you to run the v1 and v2 components in the same pipeline. Refer to this [README on specific steps](https://github.com/kubeflow/pipelines/tree/73d45eeb7136bb23f9d7e103f50a226f111c3143/components/aws/sagemaker/TrainingJob/samples/mnist-kmeans-classification)  # TODO: this links to a commit
+
+## Tutorial: SageMaker Training Pipeline for MNIST Classification with K-Means
 
 Kubeflow on AWS includes pipeline tutorials for SageMaker components. Examples are included for the most common features that can be used to run production loads with just a few clicks. To try out the examples, deploy Kubeflow on AWS on your cluster and visit the Kubeflow Dashboard `Pipelines` tab.
```

Comment:
> Examples are included for the most common features that can be used to run production loads with just a few clicks

this sentence is a bit strange, seems to suggest mnist classification using K Means is a common production workload. Can we change the tone a bit to say it includes samples about running ML workloads on SageMaker using Kubeflow pipelines ?